components/js/SlickGrid.ts

```diff
@@ -143,7 +143,7 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
 
     @Output() loadFinished: EventEmitter<void> = new EventEmitter<void>();
     @Output() editingFinished: EventEmitter<any> = new EventEmitter();
-    @Output() contextMenu: EventEmitter<{x: number, y: number}> = new EventEmitter<{x: number, y: number}>();
+    @Output() contextMenu: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
It is any event thrown by Jquery, so I don't think it can be defined. I think it depends on context.